aten/src/ATen/AccumulateType.h

```diff
@@ -31,6 +31,7 @@ template <> struct AccumulateType<char, true> { using type = int64_t; };
 template <> struct AccumulateType<int16_t, true> { using type = int64_t; };
 template <> struct AccumulateType<int32_t, true> { using type = int64_t; };
 template <> struct AccumulateType<int64_t, true> { using type = int64_t; };
+template <> struct AccumulateType<bool, true> { using type = bool; };
```

Comment:
looking at the pattern here, perhaps these should be int64_t?  Where do need these?

Reply:
(if this has to do with a different type output for accumulation) Summing over a bool mask tensor is useful to find a denominator for some masked average calculations. Currently I do sth like `sth / mask.float().sum(1)`